packages/fxa-auth-server/docs/swagger/subscriptions-api.ts

```diff
@@ -169,73 +178,123 @@ const OAUTH_MOZILLA_SUBSCRIPTIONS_CUSTOMER_PLAN_ELIGIBILITY = {
 const OAUTH_SUBSCRIPTIONS_IAP_RTDN_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/iap/rtdn',
+  notes: ['Handles a Google Play Real-time Developer Notification.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_CLIENTS_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/clients',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Returns a list of clients and their capabilities.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_INVOICE_PREVIEW_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/invoice/preview',
+  notes: ['Previews an invoice for a new plan.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_INVOICE_PREVIEW_SUBSEQUENT_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/invoice/preview-subsequent',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Preview a list of subsequent invoices.
```

Comment:
Similar to `/invoice/preview`, these include estimated tax (based on the customer's last known geolocation) and discounts.

Also worth noting that this previews these invoices based on the customer's `subscriptionId`s. That's a key difference between `/invoice/preview` and `/invoice/preview-subsequent`: the former is speculative based on the proposed plan where the user is not yet subscribed (and therefore there is no `subscriptionId`), while the latter is based on an exting subscription. See https://mozilla-hub.atlassian.net/browse/FXA-6696 .